poetapp/node
# adds env value to shell prompt

Previously, when exec'ing into the docker container and running bash,
the hostname was a docker provided, random looking string. This made it
difficult to see what container you were running inside, especially with
multiple terminals exec'ing into multiple containers.

This commit replaces the hostname with the environment variable
POET_SERVICE to make it easier to see which container you are in.

To test run:
```bash
  $ docker build -t node-test .
  $ docker run -e POET_SERVICE=node -it node-test bash
```
The shell prompt should look like this:
root@node:/usr/src/app#
